### ISSUE TITLE
README.md confusion

### ISSUE THREAD

**Issue by Hoverbear** [microsoft/WSL-DistroLauncher#62]
Hi! 

Using fresh, never opened VS2019 Community Edition installed within the last week.

When you open the `MyDistro.appxmanifest` it is unclear where the "Projects" tab is.

![image](https://user-images.githubusercontent.com/130903/66054878-80260900-e4e9-11e9-8464-840c8dc86e64.png)

Could this portion of the `README.md` receive some clarity?

**Comment by Biswa96** [comment]
In the upper left corner, File -- Edit -- View -- Project.

**Comment by Hoverbear** [comment]
@Biswa96 Sorry, I was unclear what my problem was, could you take a look at the updated description?

It's the "Packaging" tab not Project.

**Comment by Biswa96** [comment]
See this tutorial about [Package a UWP app with Visual Studio](https://docs.microsoft.com/en-us/windows/msix/package/packaging-uwp-apps) where the package tab is mentioned.

**Comment by Hoverbear** [comment]
I have already read this page (reading from https://docs.microsoft.com/en-us/windows/msix/package/packaging-uwp-apps#configure-an-app-package):

![image](https://user-images.githubusercontent.com/130903/66056533-1eb36980-e4ec-11e9-9fc2-523f0e437968.png)


Which takes you to here: https://docs.microsoft.com/en-us/windows/msix/package/create-certificate-package-signing#use-new-selfsignedcertificate-to-create-a-certificate

Which is the command I have in the issue description. :)

**Comment by Biswa96** [comment]
I don't know what is suggested by Microsoft but for personal usecase I execute `MakeCert.exe` to generate certificate and `SignTool.exe` to sign appx file. Those are in Windows 10 SDK folder.

**Comment by Hoverbear** [comment]
Ok, thanks. :)

This is still an open issue though since the README is unclear and/or incorrect.

**Comment by benhillis** [comment]
@Hoverbear - if you'd like to submit a PR to make the documentation more clear with your findings that'd be helpful.

**Comment by Hoverbear** [comment]
@benhillis I would love to, could you help me understand what the 'correct' workflow is?

**Comment by bonafont** [comment]
Change your config from ARM64 to x64, it will solve your problem. Don't know why VS open it as a ARM64 solution...

**Comment by Hoverbear** [comment]
@bonafont That sadly still does not show any packaging tab.

**Comment by gusbemacbe** [comment]
It seems that Packaging is not available for Visual Studio Community. :-(

**Comment by Hoverbear** [comment]
So it's not possible to follow the guide with VS Community?

**Comment by ZVNexus** [comment]
No, its not just an issue in Community, same thing happens in Pro. Have no idea where its located.
